api/sheets/find_dcid.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Find DCID
+nav_order: 4
```

Comment:
need to update the nav_order for all the files. They are the order appear on the ui

Reply:
Done.